src/backends/multicore/fvm.hpp

```diff
@@ -39,9 +43,12 @@ struct backend {
     using host_view   = view;
     using host_iview  = iview;
 
-    /// matrix state
-    using matrix_state =
-        nest::mc::multicore::matrix_state<value_type, size_type>;
+    using matrix_state = nest::mc::multicore::matrix_state<value_type, size_type>;
+    using multi_event_stream = nest::mc::multicore::multi_event_stream;
+
+    // re-expose common backend event types
```

Comment:
It is part of the shared API between the lowered cell and the back-end event delivery.

The `fvm_multicell` depends on and uses types and methods defined by the `backend` class, but not the other way around. Correspondingly, these have to live in the `backend` side.

The only difference with other `backend`-defined objects is that these structs are in-fact common across the `backend` implementations.